src/main/groovy/io/seqera/wave/service/persistence/impl/SurrealPersistenceService.groovy

```diff
@@ -234,4 +238,34 @@ class SurrealPersistenceService implements PersistenceService {
         return result
     }
 
+    @Override
+    void saveCondaLock(String buildId, String condaLock) {
+        final query = """\
+                                INSERT into wave_conda_lock {
+                                    buildId: '$buildId',
+                                    condaLock = '$condaLock'
```

Comment:
what if the lock file contains a `'` ?

Reply:
good point
I have not tested yet with surrealDB
I will use bytes datatype to save it